utils/copy.py

```diff
@@ -1,25 +1,41 @@
-#!/usr/bin/env python3
+#!/bin/python
+import os
+import re
+import shutil
+import argparse
 
-import os, sys, shutil
+# Instantiate the parser
+parser = argparse.ArgumentParser(description='Copy files tree from $MESON_SOURCE_ROOT to $MESON_BUILD_ROOT')
 
-copy_prefixes = sys.argv[3]
-copy_suffixes = sys.argv[4]
+# Optional positional argument
+parser.add_argument('--regex', type=str,
+                    help='Use regural expression to choose files.')
 
-# get absolute input and output paths
-input_path = os.path.join(
-    os.getenv('MESON_SOURCE_ROOT'),
-    os.getenv('MESON_SUBDIR'),
-    sys.argv[1])
+parser.add_argument('--root-directory', type=str,
```

Comment:
'help' should describe a '--root-directory' argument, not the regex one.

Reply:
Done